src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -69,6 +69,7 @@
   <ItemGroup Condition=" $(TargetFramework.Contains('-windows')) ">
     <Compile Include="**\*.windows.cs" />
     <Compile Include="**\*.windows.*.cs" />
+    <PackageReference Include="Microsoft.Toolkit.Uwp.Notifications" Version="7.1.2" />
```

Comment:
I see `UWP` on this package, do you know if this is compatible with MAUI. AFAIK MAUI will not be compatible with anything that has `UWP` just `winUI`

Reply:
starting from version 7 it supports WinUI